phoonnx/opm.py

```
@@ -62,14 +66,21 @@ def get_default_voice(self, lang: str) -> TTSModelInfo:
+                raise ValueError(f"No voices available for language: {lang}")
         return voices[0]
 
     def get_model(self, voice_id: str) -> TTSVoice:
         if voice_id in self.voices:
             return self.voices[voice_id]
         if voice_id not in self.model_manager.voices:
-            raise Exception(f"Unknown voice: {voice_id}")
+            LOG.info(f"{voice_id} not found - refreshing voice list")
+            self.refresh_voices(force=True)
+            if voice_id not in self.model_manager.voices:
+                raise Exception(f"Unknown voice: {voice_id}")
```

Ruff (0.14.3)

Create your own exception

(TRY002)